README.md

```diff
@@ -0,0 +1,133 @@
+# Action Server
+
+The action server is an actionlib server for managing the execution of high level tasks and their semantic chaining.
```

Comment:
Add a few lines what readers are expected to know, perhaps with online reference links